xml/admin_ceph_gateway.xml

```diff
@@ -1546,6 +1546,50 @@ GET /<replaceable>BUCKET</replaceable>?mdsearch
     </variablelist>
    </sect3>
   </sect2>
+
+  <sect2 xml:id="archive-sync-module">
+   <title>Archive Synchronization Module</title>
+   <para>
+    The <emphasis>archive module</emphasis> utilizes the versioning feature of
```

Comment:
archive sync module is better, like the elastic and cloud sync sections above

Reply:
updated